birdhouse/default.env

```diff
@@ -32,9 +32,12 @@ export BASH_IMAGE="bash:5.1.4"
 # Root directory under which all data persistence should be nested under
 export DATA_PERSIST_ROOT="/data"
 
+# Subdirectory containing the user workspaces
+export USER_WORKSPACES="jupyterhub_user_data"
+
 # Folder on the host to persist Jupyter user data (noteboooks, HOME settings)
 # Must use single-quote for delayed eval.
-export JUPYTERHUB_USER_DATA_DIR='${DATA_PERSIST_ROOT}/jupyterhub_user_data'
+export JUPYTERHUB_USER_DATA_DIR='${DATA_PERSIST_ROOT}/${USER_WORKSPACES}'
```

Comment:
To revert based on previous discussions.

Using `${DATA_PERSIST_ROOT}/` mount in `cowbird`/`cowbird-worker`, the link to this directory would be managed by user creation and/or permission sync with `magpie`.